homeassistant/components/homematicip_cloud/light.py

```diff
@@ -214,7 +197,7 @@ def supported_features(self) -> int:
     @property
     def unique_id(self) -> str:
         """Return a unique ID."""
-        return f"{self.__class__.__name__}_{self.post}_{self._device.id}"
+        return f"{self.__class__.__name__}_{self._post}_{self._device.id}"
```

Comment:
Is post and channel the same thing?

Reply:
No.
Post is a postfix (e.g. Battery) to a name part.
Channel is the number of the channel if the physical device has e.g. multiple sensor or switch channels.

Reply:
Won't the unique id for switch entities change then?

Reply:
The NotificationLight is special, because the two channels are named top and buttom in the native app.
I double checked that the unique_ids stay the same. Here the value of the post is used instead of the channel number.